scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/RunProcessor.java

```diff
@@ -190,4 +192,12 @@ public final Platform getPlatformType() {
    *     directory may be processed again.
    */
   public abstract NotificationDto process(File runDirectory, TimeZone tz) throws IOException;
+
+  /**
+   * Determine whether a File is readable by the processor.
+   *
+   * @param fs_obj File object which may represent a directory or file.
```

Comment:
Use camel-case and descriptive variable/parameter names. This will be a path that was specified in run scanner config, right?